Dockerfile

```diff
@@ -17,7 +17,6 @@ COPY controllers/ controllers/
 COPY cloud/ cloud/
 COPY exp/ exp/
 COPY feature/ feature/
-COPY vendor/ vendor/
```

Comment:
Is this no longer needed as part of the image build?

Reply:
yes, because the vendoring will be don by the build command, this was needed if we needed to mount the vendor